lib.bash

```diff
@@ -178,13 +178,20 @@ pool_state_color() {
   esac
 }
 
+# ZFS in FreeBSD 13 has changed slightly; `zpool status` does not
+# output a `scan` line unless a scrub has been requested.
 scrub_info() {
   local pool=$1
-
-  zpool status "$pool" \
-    | grep -A 2 '^  scan: ' \
-    | grep -E '(^        )|(scan: )' \
-    | sed -E 's/^.{8}//'
+  local info=$( zpool status "$pool" \
+              | grep -A 2 '^  scan: ' \
+              | grep -E '(^        )|(scan: )' \
+              | sed -E 's/^.{8}//' )
+
+  if [[ -z $info ]]; then
+    echo "none requested"
+  else
+    echo $info
```

Comment:
also quote here, please.

here, without the quote, you'll lose any extra whitespace since it will wordsplit and treat each word as an argument.

Reply:
Fixed in latest push.